starters/apps/basic/src/components/starter/header/header.tsx

```diff
@@ -5,28 +5,30 @@ import styles from './header.module.css';
 export default component$(() => {
   return (
     <header class={styles.header}>
-      <div class={styles.logo}>
-        <a href="/" title="qwik">
-          <QwikLogo />
-        </a>
-      </div>
-      <ul>
-        <li>
-          <a href="https://qwik.builder.io/docs/components/overview/" target="_blank">
-            Docs
-          </a>
-        </li>
-        <li>
-          <a href="https://qwik.builder.io/examples/introduction/hello-world/" target="_blank">
-            Examples
+      <div class={`container ${styles.wrapper}`}>
```

Comment:
use array here

Reply:
oh thx 🙏 updated all cases i've found (repl) 

and added a note on the styles docs:
![Bildschirmfoto_2023-04-20_um_21_27_59](https://user-images.githubusercontent.com/3241476/233468861-7e2b95ec-4f8b-482d-bc76-c45083a9eee0.png)


let me know if the wording needs an adjustment or if you have an addition

Reply:
I would have a single code snippets, not two

Reply:
![Bildschirmfoto 2023-04-21 um 13 04 58](https://user-images.githubusercontent.com/3241476/233620903-fa102aaf-1dc1-438e-9e79-c3d6347e28e2.png)

updated.